packages/location-field/src/utils.js

```diff
@@ -50,3 +50,14 @@ export const generateLabel = properties => {
 
```

Comment:
Now that there's a new method in this file, we can get rid of the eslint disable line on line 46!